sos/report/__init__.py

```diff
@@ -1333,6 +1385,24 @@ def final_work(self):
             self.policy.display_results(archive, directory, checksum,
                                         map_file=map_file)
 
+        if self.opts.estimate_mode:
+            from sos.utilities import get_human_readable
+            _sum = get_human_readable(sum(self.estimated_plugsizes.values()))
+            self.ui_log.info("Estimated disk space requirement for whole sos "
+                             "report: %s" % _sum)
```

Comment:
Let's specify that this is for an _uncompressed_ archive. If we're reporting e.g. 100MB but a user then ends up with a significantly smaller tarball, I can foresee a number of reports of inaccurate estimates and us routinely explaining this difference.

"Estimated (uncompressed) disk requirements: XXMB" maybe?

Reply:
or rather "(uncompressed) sos report directory", to be specific what is uncompressed. Ack.